src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
@SimonCropp I'm not sure I follow. Install scripts usually run after package restore had finished... so it will pull all the binaries it needs from nuget first?